tests/find_cmd_tests.rs

```diff
@@ -946,3 +946,25 @@ fn find_daystart() {
         .success()
         .stderr(predicate::str::is_empty());
 }
+
+#[test]
+#[serial(working_dir)]
+fn find_nogroup() {
```

Comment:
Why is this called `nogroup`?  There is a `-nogroup` primary, but this isn't testing it.